app/components/pools/pool-new.ts

```diff
@@ -0,0 +1,90 @@
+import {ClusterService} from '../rest/clusters';
+import {VolumeHelpers} from '';
+import {PoolService} from '';
+import {ModalHelpers} from '';
+
+interface pool {
+	pool_name: string;
+	pg_num: number;
+}
+export class PoolNewController {
+	private self = this;
+    private tier = this.tierList[0];
+	private name: string;
+	private pg_num: string;
+	static $inject: Array<string> = [
+		'$scope',
+		'$interval',
+		'$location',
+		'$log',
+		'$modal',
+		'VolumeHelpers',
+		'Modalhelpers',
```

Comment:
VolumeHelpers and ModalHelpers are not angular services. They can't be injected like this. Either the methods of theses helpers should be static or the helper should be initialized in the constructor